octobot_services/enums.py

```diff
@@ -27,4 +27,5 @@ class NotificationCategory(enum.Enum):
     GLOBAL_INFO = "global-info"
     PRICE_ALERTS = "price-alerts"
     TRADES = "trades"
+    STRATEGY_ALERTS = "strategy-alerts"
```

Comment:
What do you mean by strategy alerts ?

Reply:
alerts triggered by the "send alert ..." keyword in scripts